### PR TITLE
Add progress bar

### PR DESCRIPTION
**Pull request by rajat19**
- Changed indentation (changed indentation to 4 spaces)
- Added progress bar 
   It will show progress based on total submissions that are finalised to be printed

### CODE REVIEW

**Comment by rajatgoyal715** [comment]
@rajat19 I am getting this error. Can you please resolve this, or tell if I am executing the script wrongly.

Screenshot from 2019-10-06 16-57-02
Screenshot from 2019-10-06 16-58-40

**Comment by rajat19** [comment]
please try running it through an IDE
This import error is due to python unable to find file
In some places relative path is required
In other absolute

I would fix this so that terminal can be used

**Comment by rajat19** [comment]
Now run following
```bash
python setup.py install
python hsc/crawler.py
```

It would work now

**Comment by rajatgoyal715** [comment]
After pulling your latest changes, I tried the above-mentioned commands, but still facing the same error.
Screenshot from 2019-10-06 19-36-02

**Comment by rajatgoyal715** [comment]
@rajat19 Can we close this PR as per https://github.com/Nullifiers/Hackerrank-Solution-Crawler/issues/27#issuecomment-538750129 ?

File: setup.py [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
+		"Programming Language :: Python :: 3",
+		"License :: OSI Approved :: MIT License",
+		"Operating System :: OS Independent",
+	],
 	entry_points={
 		'console_scripts': [
 			'hsc=hsc.crawler:main',
 		],
-	}
+	}, install_requires=['requests']
```
Comment (rajatgoyal715) [inline setup.py:26]: We can also add **getpass** in this list.

Comment (rajat19) [reply setup.py:26]: ok sure

**Comment by rajat19** [comment]
Yes we can close this
I would open another PR with progress package

File: setup.py (hunk 1 above)
Comment (rajat19) [reply setup.py:26]: I think getpass is by default installed

**Pull request closed without merging**